test/Object/ObjectRest_Simple.js

```diff
@@ -0,0 +1,287 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Object Rest unit tests
+
+if (this.WScript && this.WScript.LoadScriptFile) { // Check for running in ch
+    this.WScript.LoadScriptFile("..\\UnitTestFramework\\UnitTestFramework.js");
+}
+
+var tests = [
+    {
+        name: "let assignment with simple Object",
+        body: function() {
+            let {a, b, ...rest} = {a: 1, b: 2, c: 3, d: 4};

```

Comment:
do we have tests for this scenarios?
1. multiple rests in the expression. 
2. nested rests.
3. combination of array's rest and object rest.

Reply:
1. Multiple rests is a syntax error
2. Also a syntax error (cannot have destructuring in rest) (see comment below about syntax errors)
3. Do you mean something like 
function foo(...{...rest}) {
    console.log(rest);
}

---
In reply to: [205210339](https://github.com/Microsoft/ChakraCore/pull/5498#discussion_r205210339) [](ancestors = 205210339)